src/mame/drivers/seta.cpp

```diff
@@ -9561,6 +9636,8 @@ void seta_state::thunderl(machine_config &config)
 	screen.set_screen_update(FUNC(seta_state::screen_update_seta_no_layers));
 	screen.set_palette(m_palette);
 
+	MCFG_MACHINE_START_OVERRIDE(seta_state, thunderl)
+
```

Comment:
Please don’t add more of these – we’re trying to phase them out.

Reply:
Just created the new thunderl_state class containing config for thunderl set and its bootlegs.
Also split memory maps from thunderl and wits, as the latter doesn't use the protection and the former doesn't use P3 and P4 input ports,